### ISSUE TITLE
Bug Arsenal

### ISSUE THREAD

**Issue by Philipp-00** [Wyqer/kp_liberation#976]
Bug in scripts\client\actions\open_arsenal: line 86 - Arsenal templates from the ACE Arsenal cannot be loaded because an error message appears. Error Type HashMap, expected Bool

**Comment by jishnukarri** [comment]
edit files?

**Comment by Philipp-00** [comment]
Yes, of course files were edited, but not these. There is an error in the basis of the mission.

**Comment by JasonO99** [comment]
I get this trying to load any of my ACE loadouts. On the Takistan and Lythium map at least. Just using the map from the workshop.

This only happens with the ACE Arsenal selected in the params. The issue doesn't happen with the BI Arsenals
